core/templates/pages/subtopic-viewer-page/subtopic-viewer-page.controller.ts

```diff
@@ -66,6 +68,15 @@ angular.module('oppia').directive('subtopicViewerPage', [
                   subtopicDataObject.getPageContents().getSubtitledHtml());
                 ctrl.subtopicTitle = subtopicDataObject.getSubtopicTitle();
                 PageTitleService.setPageTitle(ctrl.subtopicTitle + ' - Oppia');
+
+                let nextSubtopic = (
+                  subtopicDataObject.getNextSubtopic());
+                if (nextSubtopic) {
+                  ctrl.parentTopicId = subtopicDataObject.getParentTopicId();
```

Comment:
Why is this set, only when nextSubtopic is present? Shouldn't a back button to the topic viewer exist in all cases (I don't see that in the screenshot as well)?

Reply:
I need topic id to pass it down into subtopic-summary-tile, I believe

Reply:
Ah, ok, yeah I just checked, it is required to display the thumbnail image. Though, my second question still stands, shouldn't there be a back button to the topic viewer as seen  in the mocks?

Reply:
You are correct, I'll add that

Reply:
Done.

Reply:
Done.